cypress/e2e/copying.cy.js

```diff
@@ -1,51 +1,53 @@
-describe("Tests about copying", function () {
+describe('Tests about copying', () => {
   beforeEach(() => {
+    cy.initializeApp();
     cy.clearDebugStore();
   });
 
   afterEach(() => {
-    cy.get("[data-cy-nav-tab='testTab']").click();
-    cy.get("[data-cy-test='selectAll']").click();
-    cy.get("[data-cy-test='deleteSelected']").click();
-    cy.get("[data-cy-delete-modal='confirm']").click();
-    cy.get("[data-cy-nav-tab='debugTab']").click();
+    cy.get('[data-cy-nav-tab=\'testTab\']').click();
+    cy.get('[data-cy-test=\'selectAll\']').click();
+    cy.get('[data-cy-test=\'deleteSelected\']').click();
+    cy.get('[data-cy-delete-modal=\'confirm\']').click();
+    cy.get('[data-cy-nav-tab=\'debugTab\']').click();
   });
 
-  it("Copy report to test tab", () => {
-    cy.visit("");
-    cy.get("[data-cy-nav-tab='testTab']").click();
-    cy.get("#metadataTable tbody", { timeout: 10000 })
-      .find("tr")
-      .should("not.exist");
+  it('Copy report to test tab', () => {
+    cy.navigateToTestTabAndWait();
+    cy.get('#metadataTable tbody', { timeout: 10000 })
+      .find('tr')
+      .should('not.exist');
+    cy.log('Test1234');
```

Comment:
I think this is a log used while debugging. Can it be removed now?

Reply:
Fixed